### PR TITLE
ADD section with common scenarios

### PR DESCRIPTION
**Pull request by lukasni**
There are various scenarios that often come up in the #esi channel on slack.

This change adds a section to the docs to document some solutions to these common scenarios to prevent people from having to keep writing the same responses and hopefully act as a bit of a knowledge base for common problems.

Currently it includes documentation for two scenarios, resolving upwell structure IDs and resolving structure market orders.

### CODE REVIEW

File: docs/scenarios/resolve_structre_ids.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+### Public Structures
+
+The public `/universe/structures/` endpoint will return a flat list of IDs containing all fully public structures. *Fully public* in this context means a completely open Access Control List. If the ACL prevents anyone from docking the structure won't show up in this list.
+
+This method has the advantage of not requiring any authentication but is limited in that it only resolves completely public structures. Additionally, most endpoints that consume structure IDs require authentication anyways, so you likely won't be possible to avoid supplying a token anyways.
+
+### Corporation Structures
+
+The authenticated `/corporations/{corporation_id}/structures/` endpoit returns all structures owned by a particular corporation. It requires a token that grants the `esi-corporations.read_structures.v1` scope on a character with Director roles in the corporation.
```
Comment (ddouglas) [inline docs/scenarios/resolve_structre_ids.md:19]:
```suggestion
The authenticated `/corporations/{corporation_id}/structures/` endpoint returns all structures owned by a particular corporation. It requires a token that grants the `esi-corporations.read_structures.v1` scope on a character with the Director role in the corporation.
```

File: docs/scenarios/resolve_structre_ids.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+This endpoint provides a lot of information about the structure, including what services are installed and active, when the fuel expires, what state it currently is in etc. Be aware that it currently *does not* provide the name of the structure.
```
Comment (ddouglas) [inline docs/scenarios/resolve_structre_ids.md:21]: Are we sure about this? I can swear Zoetrope added the name to this endpoint?

File: docs/scenarios/resolve_structre_ids.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+### Other endpoints
+
+Some other endpoints may also include structure ids in their results. Structure IDs can usually be distinguished because they are in the int64 range while currently most other IDs are in the int32 range. The following endpoints will often return structure IDs:
+
+- All contract endpoints may have location_ids that are structures
+- Character and Corporation asset location_ids
+- Character Location
+- Clone locations
+- Fleet member station_id may be structure ids
+- industry jobs may be located in structure ids
```
Comment (ddouglas) [inline docs/scenarios/resolve_structre_ids.md:32]:
```suggestion
- Industry jobs may be located in structure ids
```

File: docs/scenarios/structure_markets.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+## Resolving Structure Markets
+
+Structure market orders, with the exception or ranged orders, are not included in the results of `/markets/{region_id}/orders/`. They have to be queried individually and then merged.
```
Comment (ddouglas) [inline docs/scenarios/structure_markets.md:3]:
```suggestion
Structure market orders, with the exception of ranged orders, are not included in the results of `/markets/{region_id}/orders/`. They have to be queried individually and then merged.
```

File: docs/scenarios/resolve_structre_ids.md (hunk 2 above)
Comment (lukasni) [reply docs/scenarios/resolve_structre_ids.md:21]: I had some vague memory of that too, but checking the spec shows that there is still no name included in that result.

**Pull request merged**